core/integration/llm_test.go

```diff
@@ -22,96 +24,102 @@ func TestLLM(t *testing.T) {
 	testctx.New(t, Middleware()...).RunTests(LLMSuite{})
 }
 
-func (LLMSuite) TestHelloWorld(ctx context.Context, t *testctx.T) {
-	c := connect(ctx, t)
-
-	assignment := "write a hello world program"
-	src := `
-package main
-
-import (
-	"context"
-	"dagger/test/internal/dagger"
-)
-
-type Test struct{}
-
-func (m *Test) Run(
-	assignment string,
-	model string, // +optional
-) *dagger.Container {
-	return m.llm(assignment, model).ToyWorkspace().Container()
+type LLMTestCase struct {
+	Ref   string
```

Comment:
aw man i wish i had thought of popping the examples out of the file and onto disk, it reads soooooo much better now.